ion/src/device/n0110/drivers/board.cpp

```diff
@@ -106,7 +104,7 @@ void initMPU() {
    * then an AHB error is given (AN4760). To prevent this to happen, we
    * configure the MPU to define the whole Quad-SPI addressable space as
    * strongly ordered, non-executable and not accessible. Plus, we define the
-   * Quad-SPI region corresponding to the External Chip as executable and
+   * Quad-SPI region corresponding to the Expternal Chip as executable and
```

Comment:
what is "expternal" exactly ?

Reply:
Same as the previous one, bulk copy paste mistake ^^' I developed the bootloader within the Omega 2.0 repository and I just copied paste the modified directories without checking ^^'